x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/SemanticSearchIT.java

```diff
@@ -98,195 +88,6 @@ public class SemanticSearchIT extends PyTorchModelRestTestCase {
         RAW_MODEL_SIZE = Base64.getDecoder().decode(BASE_64_ENCODED_MODEL).length;
     }
 
-    @SuppressWarnings("unchecked")
```

Comment:
It looks like all the tests have been removed from this file and the only reason it is not deleted is that `BASE_64_ENCODED_MODEL` is used in `PyTorchModelIT`.

If that is the case please move to `PyTorchModelIT` as `BASE_64_ENCODED_TEXT_EMEDDING_MODEL` along with the comment describing how it was made and delete this file.

Reply:
The reason I didn't completely remove the file is that very soon we'll have basically the same functionality available inside `_search`, and at that point we'll probably want an integration test that runs it end-to-end including the native code. `SemanticSearchIT` contains some setup functions that I thought might be useful. I am happy to adjust this PR to keep the bare minimum if you'd rather start from scratch with the semantic `_search` tests. But I thought it might save work to leave some scaffolding in place. What do you think?

Reply:
It is easy enough to restore the file from Git but it is also true that leaving the file gives me something to work with. Since the build is green this is fine with me.